import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java

```diff
@@ -56,5 +56,11 @@
   )
   boolean verbose = false;
 
+  @Parameter(
```

Comment:
Same comment here about the flag, although the pub sub project id flag isn't in this file, I think it still makes sense to label this as for stackdriver monitoring output.

Reply:
Renamed to monitoring_project_id here too.